src/BenchmarkDotNet/Environments/Runtimes/WasmRuntime.cs

```diff
@@ -24,14 +26,16 @@ public class WasmRuntime : Runtime, IEquatable<WasmRuntime>
         /// <param name="msBuildMoniker">moniker, default: "net5.0"</param>
         /// <param name="displayName">default: "Wasm"</param>
         /// <param name="aot">Specifies whether AOT or Interpreter (default) project should be generated.</param>
-        public WasmRuntime(string msBuildMoniker = "net5.0", string displayName = "Wasm", string javaScriptEngine = "v8", string javaScriptEngineArguments = "--expose_wasm", bool aot = false, RuntimeMoniker moniker = RuntimeMoniker.Wasm) : base(moniker, msBuildMoniker, displayName)
+        /// <param name="wasmDataDir">Specifies a wasm data directory surfaced as $(WasmDataDir) for the project</param>
```

Comment:
thank you for adding xml doc comment 👍 (most of our contributors ignore build warnings and don't fix them)